LICENSE

```diff
@@ -238,9 +238,6 @@ The text of each license is also included at licenses/LICENSE-[project].txt.
     (MIT License) ElasticMock v1.3.2 (https://github.com/vrcmarcos/elasticmock)
     (MIT License) MomentJS v2.24.0 (http://momentjs.com/)
     (MIT License) moment-strftime v0.5.0 (https://github.com/benjaminoakes/moment-strftime)
-    (MIT License) python-slugify v4.0.0 (https://github.com/un33k/python-slugify)
-    (MIT License) python-nvd3 v0.15.0 (https://github.com/areski/python-nvd3)
-    (MIT License) rich v9.2.0 (https://github.com/willmcgugan/rich)
```

Comment:
License for `rich` was added in https://github.com/apache/airflow/pull/12204/files but isn't needed